Static/Row.swift

```diff
@@ -134,7 +140,12 @@ public struct Row: Hashable, Equatable {
     // MARK: - Initializers
 
     public init(text: String? = nil, detailText: String? = nil, selection: Selection? = nil,
-        image: UIImage? = nil, accessory: Accessory = .None, cellClass: CellType.Type? = nil, context: Context? = nil, editActions: [EditAction] = [], UUID: String = NSUUID().UUIDString) {
+        image: UIImage? = nil, accessory: Accessory = .None, cellClass: CellType.Type? = nil, cellNib: UINib? = nil, height: CGFloat? = nil, context: Context? = nil, editActions: [EditAction] = [], UUID: String = NSUUID().UUIDString) {
+            
+        if (cellNib != nil && cellClass == nil)
```

Comment:
What do you think about changing this line to `if let nib = cellNib where cellClass == nil` and then adding the nib name to the assertion below? I think we could get two benefits out of this: 1) a more Swift-like approach, 2) more information in debugging as to _which_ cell is causing the issue.